test/test_helper.rb

```diff
@@ -13,6 +13,11 @@
 # DYNFLOW_TESTING_LOG_LEVEL = 0 # for debugging
 require 'smart_proxy_dynflow/testing'
 require 'smart_proxy_remote_execution_ssh'
+require 'smart_proxy_remote_execution_ssh/connector'
+require 'smart_proxy_remote_execution_ssh/command_update'
+require 'smart_proxy_remote_execution_ssh/dispatcher'
+require 'smart_proxy_remote_execution_ssh/command_action'
+require 'smart_proxy_remote_execution_ssh/api'
```

Comment:
Could we just use globbing to include everything? Next time we add something we might forget to put it here.